src/cloud.h

```diff
@@ -22,12 +22,17 @@
 typedef bool (*cloud_downstream_cb_t) (const char *id, struct l_queue *data,
 				       void *user_data);
 
+typedef void (*cloud_device_removed_cb_t) (const char *device_id,
```

Comment:
What do you think about to use just id instead of device_id? To keep the pattern of cloud_downstream_cb_t.

Reply:
I think device_id is more clear than id, maybe we should standardize the id as device_id.

Reply:
I agree. But will be out of the pattern. So I think that we can change to device_id after on another commit.

Reply:
Indeed.

Reply:
Done